configure.sh

```diff
@@ -24,6 +24,7 @@ export GITHUB__APP__CLIENT__SECRET=${GITHUB__APP__CLIENT__SECRET:-''}
 export GITHUB__APP__WEBHOOK__URL=${GITHUB__APP__WEBHOOK__URL:-''}
 export GITHUB__APP__WEBHOOK__SECRET=${GITHUB__APP__WEBHOOK__SECRET:-''}
 export GITHUB__APP__PRIVATE_KEY=${GITHUB__APP__PRIVATE_KEY:-''}
+export GITHUB_ORGANIZATION=${GITHUB_ORGANIZATION:-''}
```

Comment:
Not a strong opinion here, but we might want to follow the same double "_" convention we use for the rest of the variables? That said I would expect something like `GITHUB__ORG__NAME`. WDYT?